pkg/services/notification/handler.go

```diff
@@ -75,83 +79,187 @@ func (s *Server) CreateNotification(ctx context.Context, req *pb.CreateNotificat
 	if err != nil {
 		return nil, err
 	}
+	logger.Debugf(ctx, "Create Notification[%s] in DB successfully.", notification.NotificationId)
 
-	tasks, err := SplitNotificationIntoTasks(notification)
+	_, err = s.createTasksByNotification(ctx, notification)
+	if err != nil {
+		logger.Errorf(ctx, "create Tasks By Notification failed, [%+v].", err)
```

Comment:
Change the log like this: Create tasks by notification failed.

Reply:
updated.